docs/administration-guide/sso.md

```diff
@@ -1,16 +1,34 @@
 ## Setting up Single Sign-on (SSO)
 
-We recommend that you set up Single Sign-on for your Metabase installation.
+We recommend that you set up [Single Sign-on][sso-def] for your Metabase installation.
 
 ## SSO for Metabase Open Source Edition
 
-- [Google Sign-in](https://www.metabase.com/docs/latest/administration-guide/10-single-sign-on.html)
-- [LDAP](https://www.metabase.com/docs/latest/administration-guide/10-single-sign-on.html#enabling-ldap-authentication)
+- [Google Sign-in][google-sign-in]
+- [LDAP][ldap]
 
 ## SSO for Metabase paid versions
 
 With paid versions, you have more options to help manage lots of people and groups.
 
-- [JWT](https://www.metabase.com/docs/latest/enterprise-guide/authenticating-with-jwt.html)
-- [LDAP (advanced features)](https://www.metabase.com/docs/latest/administration-guide/10-single-sign-on.html#ldap-advanced-features)
-- [SAML](https://www.metabase.com/docs/latest/enterprise-guide/authenticating-with-saml.html)
+- [JWT][jwt]
+- LDAP advanced features
+  - [Group membership filter][ldap-group-membership-filter]
+  - [Syncing user attributes][ldap-user-attributes]
+- [SAML][saml]
+  - [Setting up SAML with Auth0][saml-auth0]
+  - [Setting up SAML with Google][saml-google]
+  - [Setting up SAML with Keycloak][saml-keycloak]
+  - [Okta documentation](https://developer.okta.com/docs/guides/saml-application-setup/overview/)
+  - [OneLogin documentation](https://onelogin.service-now.com/support?id=kb_article&sys_id=83f71bc3db1e9f0024c780c74b961970)
+
```

Comment:
@jeff-bruemmer 

I thought it would be helpful to include a list of common IdPs on the landing pages, but do you have any opinions about including dev docs here along with our Metabase guides?

Reply:
For cases where we don't want to (and shouldn't) take on documenting something ourselves, such as documenting third-party tooling, I'm for including links to helpful, relevant resources. We should, however, separate these resources into their own section with a heading that clearly marks them as external resources.